JXHTTP/JXHTTPOperation.h

```diff
@@ -303,4 +303,16 @@ typedef NSURLRequest * (^JXHTTPRedirectBlock)(JXHTTPOperation *operation, NSURLR
  */
 + (void)setNetworkActivityIndicatorManager:(id <JXNetworkActivityIndicatorManager>)networkActivityIndicatorManager;
 
+#if __IPHONE_OS_VERSION_MIN_REQUIRED >= __IPHONE_2_0
+
+/**
+ *  The number of active operations.
+ *  This is used to determine whether or not the activity indicator of the device is shown.
```

Comment:
This actually should be renamed since it is not _actually_ the amount of operations active - but only the amount that have the `updatesNetworkActivityIndicator` property set to `YES`. Suggestions for new names?